array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,15 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+hogwarts.forEach(({ firstName, lastName, house }) => {
+  if (house === "Gryffindor") {
+    console.log(`${firstName} ${lastName}`);
+  }
+});
```

Comment:
This is fine! You might want to separate the code that selects the people from the code that prints their names - that would make it a little easier to do something else with that list. It would also make it a little clearer which part of the object was being used for what. For example, an alternative here would be:

```js
hogwarts
  .filter(({ house }) => house === "Gryffindor")
  .forEach(({ firstName, lastName }) => {
    console.log(`${firstName} ${lastName}`);
  });
```